charts/redpanda/templates/tests/test-auditLogging.yaml

```
@@ -18,7 +18,7 @@
   This feature is gated by having a license, and it must have sasl enabled, we assume these conditions are met
   as part of setting auditLogging being enabled.
 */}}
-{{- if and .Values.auditLogging.enabled (include "redpanda-atleast-23-3-0" . | fromJson).bool }}
+{{- if and (.Values.tests.enabled) (and .Values.auditLogging.enabled (include "redpanda-atleast-23-3-0" . | fromJson).bool) }}
```

NIT:

`and` function can work with multiple arguments

 ```suggestion
{{- if and .Values.tests.enabled .Values.auditLogging.enabled (include "redpanda-atleast-23-3-0" . | fromJson).bool }}
```